plugins/outputs/timestream/timestream.go

```diff
@@ -322,25 +322,25 @@ func (t *Timestream) writeToTimestream(writeRecordsInput *timestreamwrite.WriteR
 }
 
 func (t *Timestream) logWriteToTimestreamError(err error, tableName *string) {
-	t.Log.Errorf("Failed to write to Timestream database '%s' table '%s'. Skipping metric! Error: '%s'",
+	t.Log.Errorf("Failed to write to Timestream database %q table %q: %w. Skipping metric!",
 		t.DatabaseName, *tableName, err)
 }
 
 func (t *Timestream) createTableAndRetry(writeRecordsInput *timestreamwrite.WriteRecordsInput) error {
 	if t.CreateTableIfNotExists {
 		t.Log.Infof(
-			"Trying to create table '%s' in database '%s', as 'CreateTableIfNotExists' config key is 'true'.",
+			"Trying to create table %q in database %q, as 'CreateTableIfNotExists' config key is 'true'.",
 			*writeRecordsInput.TableName,
 			t.DatabaseName,
 		)
 		err := t.createTable(writeRecordsInput.TableName)
 		if err == nil {
-			t.Log.Infof("Table '%s' in database '%s' created. Retrying writing.", *writeRecordsInput.TableName, t.DatabaseName)
+			t.Log.Infof("Table %q in database %q created. Retrying writing.", *writeRecordsInput.TableName, t.DatabaseName)
 			return t.writeToTimestream(writeRecordsInput, false)
 		}
-		t.Log.Errorf("Failed to create table '%s' in database '%s': %s. Skipping metric!", *writeRecordsInput.TableName, t.DatabaseName, err)
+		t.Log.Errorf("Failed to create table %q in database %q: %w. Skipping metric!", *writeRecordsInput.TableName, t.DatabaseName, err)
```

Comment:
`%w` should be only used for creating error, not for printing. Use `%s` or `%v` instead.